config/crd/bases/appstudio.redhat.com_releaseserviceconfigs.yaml

```diff
@@ -66,6 +66,16 @@ spec:
                       tasks
                     type: string
                 type: object
+              volumeOverrides:
+                additionalProperties:
+                  description: VolumeType is the kind of volume to be used for the
+                    managed release PipelineRun referencing this Pipeline.
+                  enum:
+                  - emptyDir
```

Comment:
We should have a value for the other type as well, no? Otherwise why have a value at all - it could just be a list of refs that use emptyDir.

Reply:
The other value is the default and it's not checked in code. Having a single value is still better than nothing as we can use a map instead of an array. It's faster. If in the future we want to support both, we could expand it.

Reply:
Resolving this as Martin is off and the PR is already approved.